src/templates/en/base.html

```diff
@@ -24,6 +24,11 @@
   You’re viewing a previous edition of the Web Almanac.
   Check out the <a href="/{{ lang }}/{{ DEFAULT_YEAR }}/{% if chapter_lang_exists(lang, DEFAULT_YEAR, slug) or page_lang_exists(lang, DEFAULT_YEAR, slug) %}{{ slug }}{% endif %}">{{ DEFAULT_YEAR }} edition!</a>
 </p>
+{%- elif year == DEFAULT_YEAR -%}
```

Comment:
This will still show after wqe publish this year's edition.

Let's add this to `src/templates/base/base.html`:

{% block call_for_contributors %}https://github.com/HTTPArchive/almanac.httparchive.org/discussions/4062{% endblock %}

And then use that to decide whether to show or not:

```suggestion
{%- elif self.call_for_contributors() | trim | length > 0 -%}
```